src/Pages/Overview/OverviewProfile.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react'
+import './App.css'
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap'
+import InfoCard from '../Profile/admin/AdminView'
```

Comment:
this line crashes the app, perhaps the route is wrong?
![image](https://user-images.githubusercontent.com/36345325/71607663-150fc100-2b30-11ea-8e02-d5408224b8c6.png)

Reply:
nothing crash on my end, travis was succeeded

Reply:
nothing crashes on my end. Travis was succeeded.

Reply:
nothing crashes on my end. Travis was succeeded.